src/screens/WyreWebview.js

```diff
@@ -50,7 +50,9 @@ export default function WyreWebview() {
       {url ? (
         <StyledWebView
           injectedJavaScript={`
-            document.getElementsByClassName('CloseBtn')[0].style.display = 'none';
+            setInterval(() => {
+              document.getElementsByClassName('CloseBtn')[0].style.display = 'none';          
+            }, 10);
```

Comment:
Is it possible that the `CloseBtn` is not found at some point for any reason? Because here we assume that there's always one available.

Also isn't running it 100 times every second a bit too much? :eyes:

Reply:
i don't this so, i don't want this button to visible even for a while. This works smoothly. maybe this will throw an error inside this webview, but is it wrong?

Reply:
As long as it doesn't stop execution of subsequent calls then I guess it's not a problem. So if we didn't observe anything wrong happening probably it's good as it is.